tests/feature/Issues/Issue1289Test.php

```diff
@@ -45,23 +45,23 @@
                 'children' => ['nope'],
             ],
         ]),
-    'default must be a string',
+    '`.0.default` must be a string',
     <<<'FULL_MESSAGE'
-    - These rules must pass for `["default": 2, "description": [], "children": ["nope"]]`
-      - Only one of these rules must pass for default
-        - default must be a string
-        - default must be a boolean
-      - description must be a string value
+    - These rules must pass for `.0`
+      - Only one of these rules must pass for `.default`
+        - `.default` must be a string
+        - `.default` must be a boolean
```

Comment:
Tests are failing here (and in other places as well). The problem is that it gets only the value, but not the path (`.default`). I think I need to change the path right before rendering the message, updating the path of the child with the parent, but I wonder if there are unintended consequences to that.

Reply:
We should reproduce the same behavior we have in the "StandardFormatter::main" into "full" and "array".

I first thought to update the path of children in "Result::withPath", but that might be weird because it looks more like a formatting concern